integrations/segmentio/lib/index.js

```diff
@@ -283,6 +283,10 @@ Segment.prototype.normalize = function(message) {
   delete msg.options;
   msg.writeKey = this.options.apiKey;
   ctx.userAgent = navigator.userAgent;
+  var locale = navigator.userLanguage || navigator.language;
+  if (!ctx.locale && locale !== undefined) {
```

Comment:
pretty unrealistic scenario, but if someone sets `ctx.locale` to `0` or `false`, `ctx.locale` will evaluate to `true`

Reply:
fixed!